functions/backup.bash

```diff
@@ -237,9 +215,156 @@ amanda_setup() {
 
       create_backup_config "${config}" "${backupuser}" "${adminmail}" "${tapes}" "${size}" AWS "${S3site}" "${S3bucket}" "${S3accesskey}" "${S3secretkey}"
     fi
+
+    whiptail --title "Operation Successful!" --msgbox "$successtext" 15 80
+  fi
+
+  echo "OK"
+}
+
+
+## "raw" copy partition using dd or mount and use rsync to sync "diff"
+## Valid arguments: "raw", "diff"
+## Periodically activated by systemd timer units (raw copy on 1st of month, rsync else)
+##
+##    mirror_SD(String method)
+##
+mirror_SD() {
+  local src
```

Comment:
Stick to one variable declaration style per function, either one line or declare and assign on different lines.

Reply:
we had that already. I use the short form for constants, longer else